mergebot.py

```diff
@@ -29,34 +27,41 @@ def main():
     successfully read in. It then waits for a keyboard interrupt as the
     signal to shut down itself and its children.
     """
-    print 'Mergebot Manager Starting Up'
+    l = logging.getLogger('mergebot')
+    log_fmt = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
```

Comment:
I think I've seen this before -- again, I'd suggest moving the logging setup into a separate class that is used in multiple places to avoid this redundancy.